doc/source/reference/commands/status.rst

```diff
@@ -147,3 +149,39 @@ values:
    * - ``version``
      - The version of the Groonga process.
      - ``5.0.7``
+
+   * - ``features``
+     - .. versionadded:: 10.0.1
+     
+       The list of Groonga's features and status(enabled or disabled).
+     - .. code-block::
+
+          {
+             "nfkc": true,
+             "mecab": true,
+             "message_pack": true,
+             "mruby": true,
+             "onigmo": true,
+             "zlib": true,
+             "lz4": false,
+             "zstandard": false,
+             "kqueue": false,
+             "epoll": true,
+             "poll": false,
+             "rapidjson": false,
+             "apache_arrow": false,
+             "xxhash": false
+          }
+  
+   * - ``apache_arrow``
+     - .. versionadded:: 10.0.1
+     
+       The version information of Apache Arrow that Groonga currently uses. Only displayed when Apache Arrow is enabled.
```

Comment:
@yoshimotoyuk Can we ensure putting a subject to all sentences?

```suggestion
       The version information of Apache Arrow that Groonga currently uses. It's only displayed when Apache Arrow is enabled.
```

Reply:
Noted. I will make sure to do so.